test/e2e/tests/tokens/nft/auto-detect-nft.spec.ts

```diff
@@ -41,9 +41,6 @@ describe('NFT detection', function () {
         await homepage.check_expectedBalanceIsDisplayed();
         await homepage.goToNftTab();
         const nftListPage = new NFTListPage(driver);
-        await nftListPage.check_nftNameIsDisplayed(
```

Comment:
nit - Do we not want to validate the name of an NFT is visible?

(happy to improve this in a separate PR)

Reply:
Another good catch: https://github.com/MetaMask/metamask-extension/pull/29445/commits/ebb829e85765486ba5b5e584ac863e7fa0bc5120